R_Core_Developers.Rmd

```diff
@@ -0,0 +1,33 @@
+# R Core Developers
+
+This page lists the former and current members of the R Core team who have write access to the R source.
+
+  * Brian Ripley (present) 
+  * Deepayan Sarkar (present) 
+  * Douglas Bates (present) 
+  * Duncan Murdoch (up to September 2017)
+  * Duncan Temple Lang (present) 
+  * Friedrich Leisch (present)
+  * Guido Masarotto (up to June 2003)
+  * Heiner Schwarte (up to October 1999)
+  * John Chambers (present)
+  * Kurt Hornik (present)
+  * Luke Tierney (present)
+  * Martin Maechler (present)
+  * Martin Morgan (present)
+  * Martyn Plummer (present)
+  * Michael Lawrence (present)
+  * Paul Murrell (present)
+  * Peter Dalgaard (present)
+  * Robert Gentleman (present)
+  * Ross Ihaka (present)
+  * Seth Falcon (up to August 2015)
+  * Simon Urbanek (present)
+  * Stefano lacus (up to July 2014)
+  * Thomas Lumley (present)
+  * Tomas Kalibera (present)
+  * Uwe Ligges (present)
+
+Affiliations of R Core team members are available [here](link to the page listing the bios most likely on the R project website--to-do). 

```

Comment:
Hi @SaranjeetKaur, This looks good. As a general tip, I would use `TODO` instead of `to-do` when you have something to add later. I believe that you can use `<!--` and `-->` to add comments in R markdown. If so, perhaps substitute:
```
...available [here]().
<!-- TODO add URL to bios, perhaps on the R project website -->
```

This way this could be merged without the specific link and if the link is clicked it will not error, it will just do nothing.

Reply:
That's a wonderful idea! Thanks!